### ISSUE TITLE
Problems running two separate Homestead environments on apple silicon + vagrant-parallels

### ISSUE THREAD

**Issue by countless-integers** [laravel/homestead#1902]
Not sure if it's an issue or something specific to my machine, but I'm getting a bit desperate and posting in hope that maybe somebody else run into a similar issue or has some experience debugging things like this.

## Versions
  - Vagrant: 2.4.0
  - Provider: Parallels
  - Homestead: 14.5.0

## Host operating system

MacOs 14.0

## Homestead.yaml

-- 

## Vagrant destroy & up output

this is last thing I see from `vagrant up`:

```
==> cic: Running provisioner: Update Composer (shell)...
    cic: Running: script: Update Composer
    cic: A connection timeout was encountered. If you intend to run Composer without connecting to the internet, run the command again prefixed with COMPOSER_DISABLE_NETWORK=1 to make Composer run in offline mode.
    cic: The following exception probably indicates you have misconfigured DNS resolver(s)
```

`vagrant destroy` seems to work ok i.e. unlike `vagrant ssh` it targets the correct box.

## Expected behavior

Provisioning and running two Homestead environments in two separate repos at the same time.

## Actual behavior

When trying to provision a second environment, while the first one is running, I get errors saying that there's no internet connection (around the time when `composer self-update` runs). Afterwards `vagrant ssh` on the first box asks me for password and then logs me into the second box. Same as vagrant ssh on the second box. Using `vagrant ssh <box id>` results in the same behaviour. `vagrant ssh-config` in both repos shows two different configs, as expected though. Halting both boxes and starting one seems to work fine.

## Steps to reproduce

1. Start one Homestead environment with Homstead being part of composer --dev dependencies (local? setup)
2. Try starting another environment in a different repo, project name, IP also using that composer setup

Not sure how to debug this one further 

## References

Parallels forums has this [thread](https://forum.parallels.com/threads/parallels-18-vagrant-multi-machine-issues.360012/) which concludes with the statement that there is/was some issue with the source "bento ubuntu" box, which sounds like it might be related to my problem.  
Maybe a new settler box version based on newer bento would help? (I don't really understand how all of those depend on each other though, so I might be talking nonsense)

**Comment by svpernova09** [comment]
Are you naming the Vagrant VMs uniquely? If not they're going to collide and weird things will happen or just plain not work.

Make sure you specify unique `name: homesteadone` and `name: homesteadtwo` in your `Homestead.yaml`

I'd have to see both configuration files and vagrant up logs to be much help.

**Comment by countless-integers** [comment]
> Are you naming the Vagrant VMs uniquely?

yes. I also have specified `ip` configuration with 2 different IPs: `192.168.56.56` and `192.168.57.57`.

I gave up on this for now. What I see happening is that both machines create 2 network interfaces for themselves: one private network and one to the shared network. From `vagrant ssh-config` I see the second machine that comes up takes over the IP of the first one. The IP itself seems to be in the shared network (checking by network adapters IP ranges in Parallels config).
I've tried removing `ip` setting, changing it to `autonetwork`, changing to IPs from the Parellels network adapter range (`10.211.55.0/24`), but none worked.
